go1.x-image/cookiecutter-aws-sam-hello-golang-lambda-image/{{cookiecutter.project_name}}/hello-world/go.mod

```diff
@@ -1,5 +1,7 @@
 require github.com/aws/aws-lambda-go v1.23.0
 
+replace gopkg.in/yaml.v2 => gopkg.in/yaml.v2 v2.2.8
```

Comment:
I assume this is required because this is a transitive dependency that's not yet updated.  Is there a mechanism to remind us to remove this line once it's no longer needed?  Or should we just override version 2.2.2 (`replace gopkg.in/yaml.v2 v2.2.2 => gopkg.in/yaml.v2 v2.2.8`) so even if we forget to remove the line, we won't be stuck on 2.2.8?